source/connect.txt

```diff
@@ -0,0 +1,200 @@
+.. _kotlin-sync-connect:
+
+==================
+Connect to MongoDB
+==================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :description: Learn how to use the Kotlin Sync driver to connect to MongoDB.
+   :keywords: client, ssl, tls, localhost
+
+.. .. toctree::
+..    :titlesonly:
+..    :maxdepth: 1
+.. 
+..    /connect/mongoclient
+..    /connect/connection-targets
+..    /connect/connection-options
+..    /connect/tls
+..    /connect/network-compression
+..    /connect/server-selection
+..    /connect/stable-api
+..    /connect/csot
+
+Overview
+--------
+
+This page contains code examples that show how to use the
+{+driver-short+} to connect your application to MongoDB by specifying
+various settings. 
+
+.. .. tip::
+.. 
+..   To learn more about the connection options on this page, see the link
+..   provided in each section.
+
+To use a connection example from this page, copy the code example into the
+:ref:`sample application <kotlin-sync-connect-sample>` or your own application.
+Be sure to replace all placeholders in the code examples, such as
+``<hostname>``, with the relevant values for your MongoDB deployment.
+
+.. _kotlin-sync-connect-sample:
+
+.. include:: /includes/usage-examples/sample-app-intro.rst
+
+.. literalinclude:: /includes/usage-examples/connect-sample-app.kt
+   :language: python
+   :copyable: true
+   :linenos:
+   :emphasize-lines: 6-8
+
+Connection
+----------
+
+The following sections describe how to connect to different targets,
+such as a local instance of MongoDB or a cloud-hosted instance on Atlas.
+
+Local Deployment
+~~~~~~~~~~~~~~~~
+
+The following code shows the connection string to connect to a local
+instance of MongoDB:
```

Comment:
Can we do intros for all the examples? Either that or removing this one to keep consistency with the other sections.

Reply:
Yep! Got caught up in the copy paste